website/static/js/fangorn.js

```diff
@@ -387,7 +387,7 @@ function checkConflicts(tb, item, folder, cb) {
                 m('p', 'Do you want to replace it?')
             ]), m('', [
                 m('span.tb-modal-btn.text-default', {onclick: cb.bind(tb, 'keep')}, 'Keep Both'),
-                m('span.tb-modal-btn.text-default', {onclick: function() {tb.modal.dismiss();}}, 'Cancel'),
+                m('span.tb-modal-btn.text-default', {onclick: function() {tb.modal.dismiss();}}, 'Cancel'), //jshint ignore:line
                 m('span.tb-modal-btn.text-defualt', {onclick: cb.bind(tb, 'replace')},'Replace'),
```

Comment:
While making the minor changes I suggested, would you mind fixing the typo here? "default"